### ISSUE TITLE
Ugly fail inserting datafile into to database

### ISSUE THREAD

**Issue by sfsinger19103** [ElectionDataAnalysis/election_data_analysis#117]
```
>>> dl.load_all()
Directory already exists: /Users/singer3/PycharmProjects/election_data_analysis/src/jurisdictions/Nevada 
Loading jurisdiction from Nevada to Engine(postgresql://singer3:***@localhost:5432/votevis_for_vv)
Directory already exists: /Users/singer3/PycharmProjects/election_data_analysis/src/jurisdictions/C:\Users\jsru2\Desktop\GitHub\election_data_analysis\src\jurisdictions\Pennsylvania 
Loading jurisdiction from C:\Users\jsru2\Desktop\GitHub\election_data_analysis\src\jurisdictions\Pennsylvania to Engine(postgresql://singer3:***@localhost:5432/votevis_for_vv)
Directory already exists: /Users/singer3/PycharmProjects/election_data_analysis/src/jurisdictions/Georgia 
Loading jurisdiction from Georgia to Engine(postgresql://singer3:***@localhost:5432/votevis_for_vv)
Processing results files ['nv18g.ini']


Processing Precinct Results 2018 General Election.csv
invalid input syntax for type integer: ""
CONTEXT:  COPY __temp_insert_singer3_20200918_135628_596960, line 1, column ReportingUnit_Id: ""

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.8/site-packages/election_data_analysis-1.0-py3.8.egg/election_data_analysis/__init__.py", line 222, in load_all
    load_error = sdl.load_results()
  File "/usr/local/lib/python3.8/site-packages/election_data_analysis-1.0-py3.8.egg/election_data_analysis/__init__.py", line 357, in load_results
    results_info, e = self.track_results()
  File "/usr/local/lib/python3.8/site-packages/election_data_analysis-1.0-py3.8.egg/election_data_analysis/__init__.py", line 343, in track_results
    e = db.insert_to_cdf_db(self.session.bind, data, "_datafile")
  File "/usr/local/lib/python3.8/site-packages/election_data_analysis-1.0-py3.8.egg/election_data_analysis/database/__init__.py", line 502, in insert_to_cdf_db
    cursor.execute(q)
psycopg2.errors.InFailedSqlTransaction: current transaction is aborted, commands ignored until end of transaction block

```

**Comment by sfsinger19103** [comment]
Problem was that there was no ReportingUnit "Nevada" in ReportingUnit.txt. This kind of error needs to cause a graceful failure.

**Comment by sfsinger19103** [comment]
Closed in #214
